src/zmq.cpp

```diff
@@ -1452,3 +1452,10 @@ int zmq_has (const char *capability_)
     //  Whatever the application asked for, we don't have
     return false;
 }
+
+void zmq_socket_monitor_pipes_stats (void *s_)
```

Comment:
For uniformity, shouldn't this return an int, and return -1 on error (ENOTSOCK/EINVAL e.g.)? What if there is no socket monitor on the socket?

Reply:
yes I forgot to deal with that - I'm adding a few more error checks and values